tvb_storage/tvb/storage/h5/encryption/data_encryption_handler.py

```diff
@@ -325,6 +324,3 @@ def run(self):
             encryption_handler.dec_queue_count(folder)
             encryption_handler.check_and_delete(folder)
             encryption_handler.sync_project_queue.task_done()
-
-
-encryption_handler = DataEncryptionHandler()
```

Comment:
I would keep this type of usage for the `encryption_handler`, any particular reason to change it? Maybe also @bvalean should review the changes related to this.

Reply:
I thought that we will only use the DataEncryptionHandler object created inside StorageInterface. It also looked a little bit confusing to import it like this.

Reply:
`DataEncryptionHandler` is intended to be used as a singleton, and in this manner, you create the object once at the first import of this module. Then you just use this object throughout the code, it is actually importing an object instead of a class, as we ussually do. Importing modules is possible only with the `import` statement, not `from .. import ..`.

Reply:
The `DataEncryptionHandlerMeta` metaclass ensures that only one instance is created

Reply:
I did this, even though I am not sure this is how you imagined it. Will push my changes very soon.